plutus-tx-plugin/test/BuiltinList/Budget/Spec.hs

```diff
@@ -19,31 +19,254 @@ tests :: TestNested
 tests =
   testNested ("BuiltinList" </> "Budget") . pure $
     testNestedGhc
-      [ goldenBundle "map" map (map `unsafeApplyCode` l)
-      , goldenBundle "elem" elem (elem `unsafeApplyCode` l)
-      , goldenBundle "find" find (find `unsafeApplyCode` l)
-      , goldenBundle "any" any (any `unsafeApplyCode` l)
-      , goldenBundle "all" all (all `unsafeApplyCode` l)
-      , goldenBundle "index" index (index `unsafeApplyCode` l)
+      [
```

Comment:
FYI: we'll have to integrate our changes together. I've updated this file in [the adjacent PR](https://github.com/IntersectMBO/plutus/pull/7113).